qiskit/quantum_info/operators/base_operator.py

```diff
@@ -636,3 +636,10 @@ def __sub__(self, other):
 
     def __neg__(self):
         return self._multiply(-1)
+
+    def __array_ufunc__(self, ufunc, method, *args, **kwargs):
+        # Dispatch numpy multiply ufunc to multiple
+        if (ufunc == np.multiply and method == '__call__') and not kwargs:
```

Comment:
Are there any other cases where numpy gets confused, or is it just calls to `np.multiply`? The list of universal functions is pretty long https://numpy.org/doc/stable/reference/ufuncs.html#available-ufuncs